oereb_einzelschutz_denkmal/update_oerebkrm_v1_1_localiseduri_table.sql

```diff
@@ -0,0 +1,14 @@
+UPDATE stage.oerebkrm_v1_1_localiseduri
+    SET atext = replace(atext, 'https://ch.so.ada.denkmal', 'https://ch.so.ada.denkmal-stage')
```

Comment:
Reicht hier nicht folgendes bereits aus:
```suggestion
    SET atext = replace(atext, 'ch.so.ada.denkmal', 'ch.so.ada.denkmal-stage')
```